test/unit_test/passes/onnx/test_conversion.py

```diff
@@ -22,3 +23,23 @@ def test_onnx_conversion_pass():
 
         # assert
         assert Path(onnx_model.model_path).exists()
+
+    # test conversion for hf model with past
+    input_model = PyTorchModel(
+        hf_config={
+            "model_name": "hf-internal-testing/tiny-random-gptj",
+            "task": "text-generation",
+            "feature": "causal-lm-with-past",
+        }
+    )
+
+    p = create_pass_from_dict(OnnxConversion, {}, disable_search=True)
```

Comment:
since this is reusing the same code with only the input model different, would you be able to parameterize the test using the input model?

Reply:
Updated. Thanks a lot!